python/lsst/analysis/tools/actions/plot/calculateRange.py

```diff
@@ -45,7 +47,10 @@ class MinMax(VectorAction):
     """
 
     def __call__(self, data: Vector, **kwargs) -> Vector:
-        return cast(Vector, [nanMin(data), nanMax(data)])
+        if len(data) == 0:
```

Comment:
Indeed, the changes from your other ticket are here, so something seems to be working 😅